### ISSUE TITLE
[bug] coverage no longer report in terminal

### ISSUE THREAD

**Issue by anewstead** [storybookjs/test-runner#412]
**Describe the bug**
not printing test report results table in the console.
test-runner `0.13.0` results print to terminal as expected 
test-runner `0.13.1-next.2` results do not print to terminal  
test-runner `0.16.0` results do not print to terminal  

**To Reproduce**
install relevant test runner and run against a sample test 
`npm test-storybook --coverage` 

**Expected behavior**
should print coverage results to terminal

**Additional context**
noticed when upgrade all storybook packages 7.4.6 > 7.6.3   
installing backward test-runner releases issue seems to have been introduced at `0.13.1-next.2`

**Comment by rfreydi** [comment]
Same observation on my project.
It look like it is a regression of a fix in the v0.12. corrected by @yannbf in https://github.com/storybookjs/test-runner/issues/332

**Comment by designbyadrian** [comment]
I also had storybook-test runner 0.16 and got no coverage report in output.
Uninstalled and installed 0.13 and now to get coverage report in output.

With 0.16, the following command(s) will generate an output to terminal:

```sh
test-storybook --coverage && nyc report --reporter=text -t coverage/storybook --report-dir .storybook
```

**Comment by yannbf** [comment]
Hey everyone! This is now fixed in version 0.18.0, sorry for the inconvenience and thanks for your patience!
